main.js

```diff
@@ -0,0 +1,184 @@
+const inputButton = document.getElementById('input-button');
+const inputValue = document.getElementById('input-value');
+const list = document.getElementById('list');
+let count = 0;
+printCount(count);
+
+inputButton.addEventListener('click', addItem);
+inputValue.addEventListener('keyup', (e) => {
+    if (e.code == "Enter") {
+        addItem();
+    }
+})
+
+//할일 조회
+$.ajax({
+    method: "GET",
+    url: "https://us-central1-pilot-todo.cloudfunctions.net/todo",
+})
+    .done(function (res) {
+        initItem(res);
+    });
+
+function initItem(res) {
+    var keys = Object.keys(res);
+    for (var i = 0; i < keys.length; i++) {
```

Comment:
C++ 같은 선언형 방식도 좋지만 JS는 함수형으로 메소드를 쓰는 게 좋아
아마 안 익숙한 방식일 것 같지만 현업에서는 대부분 함수형으로 쓴다는 거 알아두면 좋겠어
res가 만약 배열로 왔다면 나는 이렇게 했을거야

```suggestion
    res.forEach((item) => { 
```

Reply:
넵! 자바스크립트가 익숙하지 않아서 선언형 방식으로 불러왔던 것 같습니다. 함수형으로 쓰는 법도 알아보겠습니다!